src/App.tsx

```diff
@@ -1,24 +1,173 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect, useState, useCallback, FormEvent,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorType } from './types/ErrorType';
 
-const USER_ID = 0;
+export const USER_ID = 9944;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<TodoStatus>(TodoStatus.All);
+  const [isError, setIsError] = useState(false);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NoError);
+  const [isLoading, setIsLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadedTodoId, setLoadedTodoId] = useState<number[]>([]);
+
+  const getTodoList = async () => {
+    try {
+      const newTodoList = await getTodos(USER_ID);
+
+      setTodos(newTodoList);
+    } catch {
+      setIsError(true);
+      setErrorType(ErrorType.AddError);
+      setTimeout(() => {
+        setIsError(false);
+      }, 3000);
+    }
+  };
+
+  useEffect(() => {
+    getTodoList();
+  }, []);
+
+  const filteredTodos = useCallback(() => {
```

Comment:
you can use `useMemo` to save the filtered todos in variable
```suggestion
  const filteredTodos = useMemo(() => {
```